models/comments.models.js

```diff
@@ -25,3 +25,15 @@ exports.insertComment = (body, article_id, username) => {
       return comment;
     });
 };
+
+exports.removeComment = (comment_id) => {
+  return db
+    .query(`DELETE FROM comments WHERE comment_id = $1 RETURNING *`, [
+      comment_id,
+    ])
+    .then(({ rows }) => {
```

Comment:
You shouldn't need this. If you use your model that finds an individual comment and then delete it (all in your controller), then it will use the error pathway of the other model for invalid and non-existent IDs